org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessDialog.java

```diff
@@ -449,7 +449,7 @@ private void appendRecordLogInfo(ProcessInfoLog[] m_logs, HtmlBasedComponent inf
 			if (msgPresents) {
 				Td td = new Td();
 				if (log.getP_Msg() != null) {
-					if (log.getAD_Table_ID() > 0 && log.getRecord_ID() > 0) {
+					if (log.getAD_Table_ID() > 0) {
```

Comment:
this is a change of behavior that's not fully backward compatible.

Reply:
same as above - managed with the change on AEnv